src/gigs-board/entity/post/Post.jsx

```diff
@@ -474,6 +474,11 @@ const tokenMapping = {
       address: "usdt.tether-token.near",
     },
   },
+  USDC: {
+    NEP141: {
+      address: "usdc.tether-token.near",
```

Comment:
I think this is not the right usdc token address. It's possibly this one? https://nearblocks.io/token/17208628f84f5d6ad33f0da3bbbeb27ffcb398eac501a31bd6ad2011e36133a1, mentioned in [this announcement](https://www.circle.com/blog/usdc-is-now-available-on-near)

Reply:
I'm not sure, to be honest. Should I replace usdc.tether-token.near with 17208628f84f5d6ad33f0da3bbbeb27ffcb398eac501a31bd6ad2011e36133a1 ?

Reply:
Updated the address now